packages/arb-token-bridge-ui/src/util/CommonUtils.ts

```diff
@@ -16,5 +16,8 @@ export function shortenTxHash(txHash: string) {
   )}`
 }
 
+export const isE2eTestingEnvironment =
```

Comment:
Created a separate variable to avoid repetition of ` typeof window !== 'undefined' && !!window.Cypress` throughout the code.